kornia/filters/kernels.py

```diff
@@ -334,27 +339,36 @@ def get_spatial_gradient_kernel3d(mode: str, order: int, device=torch.device('cp
     return kernel
 
 
-def get_gaussian_kernel1d(kernel_size: int, sigma: float, force_even: bool = False) -> torch.Tensor:
+def get_gaussian_kernel1d(
+    kernel_size: int,
+    sigma: Union[torch.Tensor, float],
+    force_even: bool = False
+) -> torch.Tensor:
     r"""Function that returns Gaussian filter coefficients.
 
     Args:
         kernel_size: filter size. It should be odd and positive.
-        sigma: gaussian standard deviation.
+        sigma: gaussian standard deviation with shape of
+          :math:`(B, 2)`.
         force_even: overrides requirement for odd kernel size.
 
     Returns:
-        1D tensor with gaussian filter coefficients.
+        torch.Tensor: gaussian filter coefficients
```

Comment:
```suggestion
       gaussian filter coefficients
```
type is automatically added by sphinx